rules/integrations/azure/defense_evasion_azure_automation_runbook_deleted.toml

```diff
@@ -1,14 +1,14 @@
 [metadata]
 creation_date = "2020/09/01"
 maturity = "production"
-updated_date = "2021/07/20"
+updated_date = "2022/07/13"
 integration = "azure"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies when an Azure Automation runbook is deleted. An adversary may delete an Azure Automation runbook in order to
-disrupt their target's automated business operations or to remove a malicious runbook that was used for persistence.
+disrupt their target's automated business operations or to remove a malicious runbook for defense evasion.
```

Comment:
`to remove a malicious runbook ...` sounds like the customer has it installed by default and the adversary will remove it for defense evasion ?

Reply:
Good point. Change looks implemented.